processing/db_handler.py

```diff
@@ -329,8 +344,12 @@ def get_undone_data_upload_log_ac(self,
                     'ON p.log_id = o.process_id '
                     'LEFT JOIN input_interface.data_source_type s '
                     'ON u.upload_source_id = s.source_id '
+                    'LEFT JOIN '
+                    'input_interface.data_upload_type t '
+                    'ON u.upload_type_id = t.type_id '
                     'WHERE o.output_id IS NULL '
-                    'AND u.upload_type_id IN (4, 7) '
+                    'AND t.description IN (\'Half hourly data\', '
+                    '\'Half-hourly gap-filled data\') '
                     'AND s.source = %(qaqc_processor_source)s '
                     'AND log_timestamp >= '
                     'CURRENT_TIMESTAMP '
```

Comment:
Not that this is part of the same PR, but can you move "CURRENT_TIMESTAMP" to the same line on 354?